src/neptune/utils.py

```diff
@@ -16,6 +16,7 @@
 """Utility functions to support ML metadata logging with neptune.ai."""
 __all__ = ["stringify_unsupported", "stop_synchronization_callback"]
```

Comment:
Let's extend this list with your function

Reply:
Extended back 😅